index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+   
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORFOLIO</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel = "icon" href =  "/Porfolio/pictures/bxs-radiation.svg" type = "image/x-icon">
+   
+</head>
+<body>
+  <section class="prinsipal">
+    <nav class="navbar">
+      <div class="hora color" >9:45</div>
+      <img class="signal color icons" src="/Porfolio/pictures/Wifi.svg" alt="">
+      <img class="wifi color icons" src="/Porfolio/pictures/Cellular Connection.svg" alt="">
+      <img class="battery color icons" src="/Porfolio/pictures/Battery.svg" alt="">
+      <div class="name color" >John Doe</div>
+      <img class="bxicon" src="/Porfolio/pictures/bx-menu.svg" alt="">
+    </nav>
+
+    <h1 class="wilcome">Hey There. I'm Eldon</h1>
+    <div class="ocupacion">I'am a Software Developer</div>
+    <p class="parafo1">i can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you meed coded, don't hesitate to contact me.</p>
+  </section>
+<section class="s-media">
+    <div class="cat media"><img src="/Porfolio/pictures/cat.svg" alt=""></div>
+    <div class="in media"><img src="/Porfolio/pictures/linkedin.svg" alt=""></div>
+    <div class="v media"><img src="/Porfolio/pictures/victori.svg" alt=""> </div>
+    <div class="twitter media"><img src="/Porfolio/pictures/twitter2.svg" alt=""></div>
+    <div class="m media"><img src="/Porfolio/pictures/M.svg" alt=""></div>
+</section>
+
+<section class="ornament">
```

Comment:
- I think your entire line 29 to 39 will be better placed in a `<section>` tag because it represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. 🤗 

- I think line 30, `<div class="ocupacion">I'am a Software Developer</div>` will be better in an `<hx>` tag. Something like this 👉🏽 `<h2 class="ocupacion">I'am a Software Developer</h2>` because it is a text and not a container.  ✈️ 

- Currently, your social media icons are not showing on the browser. Here is what I can see compared to what's on figma 👇🏽 

| Figma Design  |Your Design |
| :---:   | :---: | 
|![good](https://user-images.githubusercontent.com/99246359/199811271-f698de76-54fe-4f96-a416-5b1ef27789ee.PNG)  |![baddd](https://user-images.githubusercontent.com/99246359/199811420-6699f132-8fb2-4b79-8bbb-39b7f81fe084.PNG)  |  

This is because your image src are not properly picking your images from your pictures folder.

You can refactor your entire line 28 to line 39 to this 👇🏽 
```
    <section>
      <h1 class="wilcome">Hey There. I'm Eldon</h1>
      <h2 class="ocupacion">I'am a Software Developer</h2>
      <p class="parafo1">i can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you meed coded, don't hesitate to contact me.</p>
      <ul class="s-media">
        <li class="cat media"><a href=""><img src="/pictures/cat.svg" alt=""></a></section>
        <li class="in media"><a href=""><img src="/pictures/linkedin.svg" alt=""></a></li>
        <li class="v media"><a href=""><img src="/pictures/victori.svg" alt=""></a></li>
        <li class="twitter media"><a href=""><img src="/pictures/twitter2.svg" alt=""></a></li>
        <li class="m media"><a href=""><img src="/pictures/M.svg" alt=""></a></li>
      </ul>
    </section>
```

- If you notice, you'd see that I refactored your social icons as a list and placed each of them in an anchor tag. This is because asides from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. ✔️ 

Kindly Note that:
1. You should re-style these changes to suit your CSS file.  🌸